src/azul/chalice.py

```diff
@@ -141,6 +141,13 @@ def __init__(self,
         self.unit_test = unit_test
         self.non_interactive_routes: set[tuple[str, str]] = set()
         assert 'paths' not in spec, 'The top-level spec must not define paths'
+        # Inject contact details into both indexer and service app Swagger specs
+        try:
+            spec['info']['description'] += config.contact_us
+        except KeyError as e:
+            # Azul unit tests also leverage the usage of this class, in those
+            # instances the Swagger spec is never configured, so we do nothing.
+            assert 'info' in e.args, 'Swagger spec is misconfigured for the AzulChaliceApp'
```

Comment:
I'd prefer not to complicate code under test with test concerns. These types of complications are better handled with fixtures and mocks.